HISTORY.md

```diff
@@ -1,3 +1,9 @@
+0.5.0 / 2020-06-04
```

Comment:
Please just put a placeholder of unreleased herw and no version number, date.

Reply:
https://www.baoding68d.net/list-games?category=